heal.js

```diff
@@ -25,11 +25,21 @@ var ClusterManager = require('./lib/cluster.js');
 var assertNoError = require('./lib/util.js').assertNoError;
 var program = require('commander');
 
+function safeParseInt(number, defaultValue) {
+    number = parseInt(number, 10);
+    if (isNaN(number)) {
+        return defaultValue;
+    } else {
+        return number;
+    }
+}
+
 function main() {
     program
         .description('Start a partition heal coordinated by the target node')
         .option('--tchannel-v1')
-        .option('--verbose')
+        .option('--quiet')
```

Comment:
Makes sense. I only ask, because it would be easy for the operator to silence the status output by doing `>/dev/null`. Then we don't need to maintain a `--quiet` switch.